docs: Landing page and Logo Update

dat-adi opened the pull request:
## Changelog
- [x] Improved landing page.
- [x] Addition of the new ForgeFlux Logo into the Documentation.

dat-adi committed:
Updated the introduction page

dat-adi committed:
Updated the landing page

Set up a new landing page. The goal is to diss on forge-hopping, and
have people despair about the numerous operations that they have to make
when hopping.

realaravinth commented on content/_index.md:
```diff
@@ -3,43 +3,53 @@ title = "Magic Bridges"
 
-[ForgeFlux](https://forgeflux.org/) aims to implement federation in [software forges](https://en.wikipedia.org/wiki/Forge_(software)) , enabling
-a decentralized development environment.
+It's a Friday evening, and you've sat down to work on some code. \
```
Can you remove `\` from all these documents?

dat-adi replied:
That might result in extremely large paragraphs, we cool with that?

The backslashes function as reasonable line breakers.

realaravinth reviewed (changes requested):
`\` seems to be there at the end of all sentences and it seems to [creep into the rendered content](https://github.com/dat-adi/docs/blob/main/content/_index.md)

dat-adi committed:
Removed backslashes

Condensed the content into paragraphs instead of sentences with a
newline.

dat-adi committed:
Terse content

Reworked the content to be a bit more polished.

realaravinth commented on config.toml:
```diff
@@ -8,6 +8,6 @@ highlight_code = true
 
 [extra]
-logo = "https://easydocs.codeandmedia.com/logo.svg"
+logo = "https://github.com/forgeflux-org/docs/blob/main/static/logo.svg"
```
please change this URL to `/logo.svg`, all files in `/static` dir are available at root

dat-adi replied:
Ah, very well. That will be done.

realaravinth commented on static/logo.svg:
```diff
@@ -1 +1 @@
-<svg id="logo-11" width="119" height="30" viewBox="0 0 119 30" fill="none" xmlns="http://www.w3.org/2000/svg"> <path d="M2.94098 27.825H28.591V2.175H2.94098V27.825ZM23.654 12.532H18.234V7.112H23.654V12.532ZM7.87698 7.112H13.3V17.468H23.654V22.888H7.87698V7.112ZM45.294 2.175C42.7574 2.175 40.2778 2.92718 38.1688 4.33641C36.0597 5.74564 34.4159 7.74863 33.4452 10.0921C32.4745 12.4356 32.2206 15.014 ...
\ No newline at end of file
+<svg id="a03ed06d-885d-4bed-8a4a-44974750f5a8" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 501.34 185.17"><defs><style>.a7dc26de-4b77-4d05-9982-f373222849ac,.be9c742b-a6b1-41a4-97ac-f5e00be0ef3e{fill:#ee362d;}.ab3dba79-30e8-459a-a0be-ce4217101439{fill:#6d6f6e;}.a7dc26de-4b77-4d05-9982-f373222849ac{font-size:54.24px;font-family:Montserrat-Medium, Montserrat;font-weight:500; ...
```
The logo styling changes in different zoom levels, can you contact Sai Pranay and get this fixed?

https://user-images.githubusercontent.com/33521410/150632830-311dd2db-64d0-4a08-9420-ca9fc459594b.mp4

dat-adi replied:
Interesting. Sure, I'll get in touch with him.

dat-adi replied:
The issue has been resolved.

dat-adi committed:
Updated config and SVG

The logo now points to the root directory, and the SVG has been replaced
after resolving the issue with the viewport.

dat-adi commented:
The sidebar is now displayed with an appropriately sized logo as compared to the initial draft.
![image](https://user-images.githubusercontent.com/31721284/150685473-3219a52d-2632-4ee9-88e9-873f458efe94.png)
